src/view_control.cpp

```diff
@@ -616,7 +616,7 @@ void View::DrawOctave(
     data_OCTAVE_DIS dis = octave->GetDis();
     data_STAFFREL_basic disPlace = octave->GetDisPlace();
 
-    int y1 = octave->GetDrawingY();
+    const int y1 = octave->GetDrawingY();
```

Comment:
Using const is OK, but why?  Const enforces that the variable should not be changed, and there should be an important reason for it not to change which I am not seeing here.  If that is what you want to do, then const is appropriate, and if so, then a comment explaining why it should not change would be useful to add to the code.

Reply:
correct again. Now it's constant, but we might want to refine it later.